archimedes/Gemfile

```diff
@@ -1,7 +1,7 @@
 source 'https://rubygems.org'
 
 # etna application/server gem
-gem 'etna'
+gem 'etna', git: 'https://github.com/mountetna/monoetna.git', ref: ''
```

Comment:
Going to remove this, was an aborted attempt to modify archimedes, but I decided to scope changes to just `timur` `metis` `janus` and `magma` for now.